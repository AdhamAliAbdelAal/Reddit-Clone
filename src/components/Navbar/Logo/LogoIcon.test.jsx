import renderer from 'react-test-renderer';
import LogoIcon from './LogoIcon';

test('test snapshot', async () => {
  const tree = renderer.create(<LogoIcon />).toJSON();
  expect(tree).toMatchSnapshot();
});
