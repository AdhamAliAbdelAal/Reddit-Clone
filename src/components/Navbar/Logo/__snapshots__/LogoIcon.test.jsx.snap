// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test snapshot 1`] = `
<a
  href="wwww.reddit.com"
>
  <div
    className="MuiBox-root css-k008qs"
  >
    <div
      className="MuiBox-root css-8opyo5"
    >
      <img
        alt=""
        height={32}
        src="Logo.svg"
        width={32}
      />
    </div>
    <div
      className="MuiBox-root css-tjofhy"
    >
      <img
        alt=""
        height={20}
        src="LogoText.svg"
        width={57}
      />
    </div>
  </div>
</a>
`;
